docs/data-product-studio/data-quality/snowplow-inspector/adding-schemas/index.md

```diff
@@ -69,12 +69,12 @@ These schemas are managed within the extension and stored on your own machine.
 The only required configuration is a name, and the schemas themselves.
 
 ## Data Structures API registries
-This type is recommended for use with Snowplow BDP via the [Data Structures API](/docs/data-product-studio/data-structures/manage/api/index.md).
+This type is recommended for use with the [Data Structures API](/docs/data-product-studio/data-structures/manage/api/index.md).
 
 In order to function, the extension requires:
 
-- Organization ID: This is usually found in the URL when logged into the Snowplow BDP console. See more at [Managing Console API authentication](/docs/account-management/index.md#version-2).
-- API Key: When logged into the Snowplow BDP console, should be available in [API keys for managing Snowplow](https://console.snowplowanalytics.com/credentials) (within "Manage organization"). See more at [Managing Console API authentication](/docs/account-management/index.md#version-2).
+- Organization ID: this is usually found in the URL when logged into Snowplow Console. See more at [Managing Console API authentication](/docs/account-management/index.md#version-2).
```

Comment:
If you are updating this line, might as well link to where this now lives: https://console.snowplowanalytics.com/settings